examples/windows_nodepool_with_private_cluster/gke/main.tf

```diff
@@ -0,0 +1,95 @@
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "gke_private_cluster" {
+  source                     = "../modules/google_kubernetes_engine/gke_private_cluster/"
```

Comment:
nit: is this a custom module?

Reply:
We should not need anything in `./examples/windows_nodepool_with_private_cluster/*`. We can use for instance `"../../modules/beta-private-cluster/"`.

Reply:
Yes @bharathkkb  its custom model has taken reference from this same repo, just modified settings to deploy windows nodepool on GKE cluster